src/Integration.Vsix/SonarLintTagger/IssueMarker.cs

```diff
@@ -33,8 +36,11 @@ namespace SonarLint.VisualStudio.Integration.Vsix
     /// </remarks>
     internal class IssueMarker : IFilterableIssue
     {
-        public IAnalysisIssue Issue { get; }
-        public SnapshotSpan Span { get; }
+        public IAnalysisIssueVisualization IssueViz { get; }
+
+        public IAnalysisIssue Issue => IssueViz.Issue;
+
+        public SnapshotSpan Span => IssueViz.Span.Value;
```

Comment:
Check if it has value?

Reply:
There didn't seem to be much point doing any special here since the whole class is going to be removed.